srcpkgs/autossh/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'autossh'
 pkgname=autossh
-version=1.4f
+version=1.4g
 revision=1
 build_style=gnu-configure
+hostmakedepends="openssh"
 short_desc="Automatically restart SSH sessions and tunnels"
 maintainer="uriahheep <uriahheep@gmail.com>"
 license="MIT"
-hostmakedepends="openssh"
-homepage="http://www.harding.motd.ca/autossh/"
-distfiles="http://www.harding.motd.ca/autossh/autossh-$version.tgz"
-checksum=0172e5e1bea40c642e0ef025334be3aadd4ff3b4d62c0b177ed88a8384e2f8f2
+homepage="https://www.harding.motd.ca/autossh/"
+distfiles="${homepage}/autossh-$version.tgz"
```

Comment:
Why using variable substitution here? Is homepage going to vary?

Reply:
Saw some other packages doing that. Is there a policy for this case?

Reply:
It isn't obligatory i do when the distfiles= line gets too long